zallet/src/config.rs

```diff
@@ -282,6 +288,115 @@ impl LimitsSection {
     }
 }
 
+/// Note management configuration section.
+///
+/// TODO: Decide whether this should be part of `[builder]`.
+#[derive(Clone, Debug, Default, Deserialize, Serialize, Documented, DocumentedFields)]
+#[serde(deny_unknown_fields)]
+pub struct NoteManagementSection {
+    /// The minimum value that Zallet should target for each shielded note in the wallet.
+    pub min_note_value: Option<u32>,
```

Comment:
I'd like to think of a better name for this field; it's a little bit ambiguous. Maybe `min_managed_note_value`?